### PR TITLE
add `flextables` extension to R v4.2.1 + v4.2.2

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/r/R/R-4.2.2-foss-2022b.eb [hunk 1]
```diff
@@ -78,9 +78,6 @@ exts_default_options = {
 # !! order of packages is important !!
 # packages updated on 17th March 2023
 exts_list = [
-    # include packages that are part of the base installation of R,
-    # both to make sure they are available (via sanity check),
-    # and to be able to pass the check for required dependencies when installing extensions in parallel
```
Comment (boegel) [inline easybuild/easyconfigs/r/R/R-4.2.2-foss-2022b.eb:83]: @verdurin Let's retain these comments?

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/r/R/R-4.2.2-foss-2022b.eb (hunk 1 above)
Comment (verdurin) [reply easybuild/easyconfigs/r/R/R-4.2.2-foss-2022b.eb]: Restored.

Comment (verdurin) [reply easybuild/easyconfigs/r/R/R-4.2.2-foss-2022b.eb]: I guess it's the `checksum` script that stripped them.

Comment (boegel) [reply easybuild/easyconfigs/r/R/R-4.2.2-foss-2022b.eb]: Yeah, it's `--inject-checksums`, see also https://github.com/easybuilders/easybuild-framework/issues/3269

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2
EB_ARGS="--skip"

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3109.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/f75e306b281d128649f04afa68f754f3 for a full test report.

**Review by boegel — APPROVED** [review]

**Pull request merged**